driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -637,7 +637,7 @@ public void onSet(Connection connection, Message.Response response, long latency
                                                 + "Seeing this message a few times is fine, but seeing it a lot may be source of performance problems",
```

Comment:
Need to add special case at line 626 to allow case where keyspaces don't match if protocol version >= 5